schedule.md

```diff
@@ -0,0 +1,89 @@
+# Session plan
+
+## Session 1: R and statistical concepts used
+
+Some of this could be pre-course reading material, see e.g. https://sbfnk.github.io/mfiidd/slides/rintro.pdf
```

Comment:
I can make some stan precourse material but I think we will still want a 15 minute review lecture of what it is.

Reply:
I've made #22 and #21 and assigned myself for this